source/textUtils.py

```diff
@@ -12,15 +12,102 @@
 import sys
 import ctypes
 from collections.abc import ByteString
-from typing import Tuple, Optional
+from typing import Tuple, Optional, Type
 import locale
 from logHandler import log
+from abc import abstractmethod
 
 WCHAR_ENCODING = "utf_16_le"
+UTF8_ENCODING = "utf-8"
 USER_ANSI_CODE_PAGE = locale.getpreferredencoding()
 
 
-class WideStringOffsetConverter:
+class OffsetConverter:
+	decoded: str
+	
+	def __init__(self, text: str):
+		if not isinstance(text, str):
+			raise TypeError("Value must be of type str")
+		self.decoded: str = text
+
+	def __repr__(self):
+		return "{}({})".format(self.__class__.__name__, repr(self.decoded))

```

Comment:
Can't you use f-string here?

Reply:
I'm just refactoring - that's legacy code.